docs/articles/pkgdown/replication/text2vec.Rmd

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Why is this whole file showing up in the diff as an addition? I can't tell what was changed in it.